kythe/go/serving/xrefs/xrefs.go

```diff
@@ -1403,7 +1403,7 @@ func (s *refStats) addAnchors(to *[]*xpb.CrossReferencesReply_RelatedAnchor, grp
 		firstNonEmpty := len(scopedRefs)
 		for i := 0; i < len(scopedRefs) && s.skip > 0; i++ {
 			sr := scopedRefs[i]
-			if len(sr.GetReference()) > s.skip {
```

Comment:
Is it straightforward to add a test of this?

Reply:
No, I'll look into it in another PR.